src/explore-education-statistics-common/src/modules/table-tool/components/FiltersForm.tsx

```diff
@@ -36,7 +42,14 @@ interface Props {
     indicators: string[];
     filters: string[];
   };
+  selectedPublication?: SelectedPublication;
+  subject: Subject;
   subjectMeta: SubjectMeta;
+  tableSizeErrorDownloadAvailable?: boolean;
+  tableSizeErrorLogEvent?: (
```

Comment:
This is an event handler so we should name this according to those conventions i.e. `onTableSizeError`

Reply:
Happy that this has been addressed :)